src/main/java/com/echobox/api/linkedin/client/DefaultWebRequestor.java

```diff
@@ -369,6 +367,7 @@ public Response executePut(String url, String parameters, String jsonBody,
       }
     }
   }
+  // CPD-ON
```

Comment:
Probably means that some of the logic in the two methods should be put in a common method. Don't need to do that here though.

Reply:
There was one particular line that it was complaining about, which didn't even seem like a duplicate. I didn't want to spend too much time investigating so escaped the duplication